NEWS.rst

```diff
@@ -1,4 +1,4 @@
-This file contains the release notes for the Twisted.
```

Comment:
… how long has that been there? :)

Reply:
I will let the historian to write a story about it :p

thanks for the review.

I will trigger the final release